svf-llvm/lib/LLVMModule.cpp

```diff
@@ -298,9 +300,7 @@ void LLVMModuleSet::createSVFFunction(const Function* func)
 
     for (const BasicBlock& bb : *func)
     {
-        SVFBasicBlock* svfBB =
-            new SVFBasicBlock(getSVFType(bb.getType()), svfFunc);
-        svfFunc->addBasicBlock(svfBB);
+        SVFBasicBlock* svfBB = bbGraph->addBasicBlock(bb.getName().str());
```

Comment:
put this line "SVFBasicBlock* svfBB = bbGraph->addBasicBlock(bb.getName().str());" into "addBasicBlockMap" and rename addBasicBlockMap to addBasicBlock